--- sources/openjdk8/common/autoconf/build-performance.m4.orig	2016-12-11 06:09:30.000000000 -0500
+++ sources/openjdk8/common/autoconf/build-performance.m4	2016-12-11 06:10:23.000000000 -0500
@@ -199,7 +199,7 @@
     # Only use ccache if it is 3.1.4 or later, which supports
     # precompiled headers.
     AC_MSG_CHECKING([if ccache supports precompiled headers])
-    HAS_GOOD_CCACHE=`($CCACHE --version | head -n 1 | grep -E 3.1.@<:@456789@:>@) 2> /dev/null`
+    HAS_GOOD_CCACHE=`($CCACHE --version | head -n 1) 2> /dev/null`
     if test "x$HAS_GOOD_CCACHE" = x; then
       AC_MSG_RESULT([no, disabling ccache])
       CCACHE=
